.github/dependabot.yaml

```diff
@@ -1,6 +1,24 @@
 ---
 version: 2
 updates:
+  - package-ecosystem: "gomod"
+    directory: "/"
+    labels:
+      - "dependencies"
+      - "go"
+      - "type::chore"
+    schedule:
+      interval: "weekly"
+
+  - package-ecosystem: "gomod"
+    directory: "/kurlkinds"
```

Comment:
This directory does not exist. This seems like it's copied from another repo.

Reply:
Ops, good catcher.